internal/server/server_ingest.go

```diff
@@ -34,7 +35,7 @@ func (s *Server) Ingest(ctx context.Context, request *talaria.IngestRequest) (*t
 		}
 
 		// Partition the request for the table
-		blocks, err := block.FromRequestBy(request, appender.HashBy(), filter, s.computed...)
+		blocks, err := block.FromRequestBy(request, appender.HashBy(), filter, block.Transform(filter, s.computed...), stream.Publish(s.streams, s.monitor))
```

Comment:
@TiewKH @tardunge I just realized after merging, but `streams` should be per-table, similar to the compaction. At the moment, if there's 2+ tables, we stream the same thing multiple times to the same topics.